setup.py

```diff
@@ -27,8 +27,8 @@
         'pyBigWig==0.3.4',
         'numpy==1.11.2',
         'httpretty==0.9.5',
-        'pycellbase==0.3.3'
-        'pypanelapp==1.0.3'
+        'pycellbase==0.3.3',
```

Comment:
this pycellbase looks terribly old - could/should we update to 4.7.1 (current) ?

Reply:
@javild  I've bumped it up to 4.7.0 as that's what the bertha runtime environment has, so that's the version we've been using already. Are there decent differences between 4.7.0 and 4.7.1?

Reply:
no there aren't, 4.7.0 is fine